shared/react-native/ios/Keybase/AppDelegate.m

```diff
@@ -43,6 +45,19 @@ - (BOOL)addSkipBackupAttributeToItemAtPath:(NSString *) filePathString
   return success;
 }
 
+- (void) createBkgReadableDir:(NSString*) path
```

Comment:
OK, the docs seemed to imply the attributes were for files, not directories... like does it check the parent directory attributes or something?

I guess as long as it works...